### PR TITLE
Added functions for like button and comment button to interact with database and added meetings minutes.

### PR DESCRIPTION
**Pull request by moiikram**
- Added functions for like button to interact with database but currently does not work. 

- added the meeting minutes till date.

### CODE REVIEW

**Comment by alexsenden** [comment]
Lets merge this after the sprint evaluation, we already have everything we need for this sprint and this alone will not finish any user story.

File: harmony-api/src/routes/post/postRouter.ts
Comment (Dankgen) [inline harmony-api/src/routes/post/postRouter.ts:1]: Seems like an accidental duplication of: 
` router.get('/:postId', postController.getPost) `

File: harmony-api/src/controllers/post/postController.ts
Comment (Dankgen) [inline harmony-api/src/controllers/post/postController.ts:1]: If its not too hard for you to understand,
Would you be willing to switch out the  `userId` with `userCookie`
Then get the user from the cookie. I just suggest this for some security.

File: harmony-api/package.json
Comment (Dankgen) [inline harmony-api/package.json:1]: Any reason for moving `prisma` to a dev dependency?

File: harmony-api/src/controllers/post/postController.ts
Comment (Dankgen) [reply harmony-api/src/controllers/post/postController.ts:1]: Sorry not leaving too much of an explanation
In the front end, you can acquire this with `const userCookie = useContext(UserCookieContext)`
In the backend you can acquire the user with `const user = userService.getUserFromCookie(userCookie)`

**Review by alexsenden — APPROVED** [review]
Looks good! Very clean!

**Pull request merged**